stats/test/platform-routes.test.js

```diff
@@ -344,21 +344,24 @@ const givenDailyStationMetrics = async (pgPoolEvaluate, day, stationStats) => {
       station_id,
       participant_address,
       inet_group,
-      accepted_measurement_count
+      accepted_measurement_count,
+      total_measurement_count
```

Comment:
```
  1) Platform Routes HTTP request handler
       GET /stations/daily
         returns daily station metrics for the given date range:
     error: null value in column "total_measurement_count" of relation "daily_stations" violates not-null constraint
      at /home/runner/work/spark-stats/spark-stats/node_modules/pg-pool/index.js:45:11
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async givenDailyStationMetrics (file:///home/runner/work/spark-stats/spark-stats/stats/test/platform-routes.test.js:341:3)
      at async Context.<anonymous> (file:///home/runner/work/spark-stats/spark-stats/stats/test/platform-routes.test.js:56:7)
```

I think it will be best to keep backwards compatibility and assume `total_measurement_count = accepted_measurement_count` when the caller does not provide the last argument.

Reply:
I had started working on a fix before you suggested this, please tell me if you would like to change the solution

Reply:
My proposal would require fewer code changes (it would create less code churn), but I am fine either way.